paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -148,6 +148,19 @@ class PaymentSheet internal constructor(
          */
         val allowsDelayedPaymentMethods: Boolean = false,
 
+        /**
+         * If true, allows payment methods that require a shipping address, like Afterpay and
+         * Affirm. Defaults to false.
+         *
+         * Set this to true if you collect shipping addresses and set [shippingDetails] or set
+         * `shipping` details directly on the PaymentIntent.
```

Comment:
Should this be more explicit?

```suggestion
         * Set this to true if you collect shipping addresses and set
         * [PaymentSheet.FlowController.shippingDetails] or set `shipping` details directly on the
         * PaymentIntent.
```

Reply:
Should we mention all three? `shippingDetails` in `PaymentSheet.Configuration`, `shippingDetails` in `PaymentSheet.FlowController.shippingDetails`, or `shipping` in `PaymentIntent`